src/votes/models.py

```diff
@@ -1,3 +1,32 @@
 from django.db import models
 
-# Create your models here.
+from main.models import Entity
+from case_gather.models import Case
+from parliament.models import ParliamentMember
+
+class VoteRecord(Entity):
+    '''
+    The voting record of each case
+    '''
+    case = models.ForeignKey(Case)
+    althingi_id = models.IntegerField()
+    '''
+    Number of people who voted yes
+    '''
+    yes = models.IntegerField(null=True, blank=True)
+    '''
+    Number of people who voted no
+    '''
+    no = models.IntegerField(null=True, blank=True)
+    '''
+    Number of people who did not vote
+    '''
+    didNotVote = models.IntegerField(null=True, blank=True)
```

Comment:
I guess the _more_ correct word is 'abstain'

Reply:
mmmmmmmmmmmmmmmmmm I guess!